packages/core/crates/core-crypto/src/checksum.rs

```diff
@@ -0,0 +1,90 @@
+use sha3::{digest::DynDigest, Keccak256};
+use utils_types::primitives::Address;
+
+// https://eips.ethereum.org/EIPS/eip-55
+pub fn to_checksum(address: Address) -> String {
+    let address_hex = address._to_hex();
```

Comment:
Please use the `to_hex` function without the underscore. Those ones are meant for JS

Reply:
I think linter didn't recognize `to_hex()` for some reason and failed to compile. What did I do wrong there?

Reply:
Oh found the missing piece : forgot to `use utils_types::traits::ToHex`